README.md

```diff
@@ -9,29 +9,32 @@ A secure and easy-to-use password manager for macOS and Linux.
 
 ## Features
 
-* Strong, configurable modern encryption with sensible defaults
-* Configurable backend storage
-* Password generator
+* Strong, modern encryption with sensible defaults (ChaCha20-Poly1305, AES-GCM, OpenPGP)
+* Configurable safe storage (file, AWS S3)
 * Secure notes
+* Built-in password generator
 * Clipboard support
-* Sync support
 * Automatic backups
-* Export to JSON
+* Sync multiple safes
+* Export accounts to JSON
 * No external dependencies
 
 ## Install
+**go get**
```

Comment:
Why do you prefer bold texts instead of headlines? Using headlines we can link to the specific section on Github directly (e.g. ```[Install via Homebrew](###homebrew)```)